### ISSUE TITLE
How to remove red lines at the edge of screen when i entered Settings?

### ISSUE THREAD

**Issue by ChoiWooYoung** [ungoogled-software/ungoogled-chromium-android#24]
How can i disable this red lines?
I thought that i can disable it at Accessibility but, I cannot find the setting that disables red lines.

Anyway, Thanks for a good project :D

![Screenshot_20200728-153203_Ungoogled Chromium Extensions.jpg](https://user-images.githubusercontent.com/11593330/88627803-deab1c00-d0e7-11ea-8e9d-7d6d2148359b.jpg)

**Comment by ChoiWooYoung** [comment]
Device: Samsung Galaxy Note 8
Android Version: 9 Pie ARM64
Chromium Version: Monochrome 84.0.4147.89

**Comment by wchen342** [comment]
Duplicate of [#53@Gitea](https://git.droidware.info/wchen342/ungoogled-chromium-android/issues/53). To put it simple, it's the strict mode check enabled by Chormium internally. It is just a warning and has no harm.
Can you check whether the same happens to unmodified Chromium of the same version? If it also happens then please file an issue upsteam.

**Comment by ChoiWooYoung** [comment]
Yes, Strict mode was disabled and also, Original Chromium is same as this too. I think Chromium internally automarically enables Strict mode in app

**Comment by wchen342** [comment]
Yes, they have a lot of places that uses strict mode check in their codes. It is not really feasible for me to remove all of them, so the only way is basically either asking Chromium dev to disable this or just ignore them since they are harmless.

**Comment by abbietapmer** [comment]
Hey, on [#53](https://git.droidware.info/wchen342/ungoogled-chromium-android/issues/53),  I made [a bug report](https://bugs.chromium.org/p/chromium/issues/detail?id=1111908) on Chromium's bugs page and I was told to install the Canary version instead, which had no 'red border' problems.

**Comment by wchen342** [comment]
Isn't the canary version way past v84? I think it is supposed to be a nightly version.
Also I didn't build a debug version, `is_debug` is set to `false` so that shouldn't be the problem.

**Comment by abbietapmer** [comment]
Stable is on v84 and for Canary (Nightly) it is v86. The issue doesn't persist on Canary but I have it on UC.

**Comment by wchen342** [comment]
If the problem occurs on v84 and disappears on v86 then probably they fixed it during v85~v86? Maybe should wait a little longer to see whether in next version the problem will solve itself.

**Comment by wchen342** [comment]
See update at [https://git.droidware.info/wchen342/ungoogled-chromium-android/issues/53#issuecomment-447](https://git.droidware.info/wchen342/ungoogled-chromium-android/issues/53#issuecomment-447). I'd like to hear opinions.

**Comment by wchen342** [comment]
Should be fix by 1812d02.
